docker-compose.dev.yml

```diff
@@ -58,7 +58,7 @@ services:
       - CELERY_RESULT_BACKEND=redis://redis:6379/0
     restart: always
     volumes:
-      - resin-data:/data
```

Comment:
I'm still not sure about this. It should probably be balena-data. Please check on the balena forum how this will be migrated for existing installations.